angular 2 app submit

rahullsinha opened the pull request.

lpolepeddi commented:
@rahullsinha I'm seeing this when I visit https://extradiscount.in.

![screen shot 2017-02-21 at 4 14 19 pm](https://cloud.githubusercontent.com/assets/2704391/23185282/eae0c236-f850-11e6-913f-bb34b32ed9ae.png)


Is the URL correct?

rahullsinha commented:
we have fixed the issue. there is some problem with firebase hosting so we switched to another hosting provider.
you can check here[https://geopeeker.com/fetch/?url=https%3A%2F%2Fextradiscount.in%2F](url)

lpolepeddi commented:
Thanks! ExtraDiscount is up on Made with Angular - https://www.madewithangular.com/sites/extradiscount
